src/main/java/world/bentobox/challenges/database/object/Challenge.java

```diff
@@ -311,7 +311,7 @@ public boolean isDeployed()
      */
     public ItemStack getIcon()
     {
-        return icon.clone();
+        return icon !=null ? icon.clone() : new ItemStack(Material.PAPER);
```

Comment:
I found this would cause an NPE if for some reason the icon was null. It occurred once during an import of some ASkyBlock file and so I added this code to defend against it. It's not related to database transition code.